### ISSUE TITLE
Local images?

### ISSUE THREAD

**Issue by Wicked-Mouse** [perceptron8/pinky.ext#27]
Not really an issue, but rather a request: would it be possible to add a part where the script checks whether a local version of the image files is in the add-on folder as a replacement, instead of the fireball image that generates the magenta image? I'd like to be able to create my own versions of the images if possible, so that it's still easy to find them but not with the magenta squares  =)

**Comment by perceptron8** [comment]
This is duplicate of #1. To summarize, Flash plugin doesn't handle [data urls](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs), [blob urls](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL), [runtime urls](https://developer.chrome.com/extensions/runtime#method-getURL). It's possible to redirect to images on another domain (including local web servers), but this doesn't always work as expected (see #5 and #11). You can experiment with [switcheroo-redirector](https://chrome.google.com/webstore/detail/switcheroo-redirector/cnmciclhnghalnpfhhleggldniplelbg) and [web-server-for-chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb) (note that [Chrome Apps are now deprecated](https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html)) to achieve your goal. See [background.js](https://github.com/perceptron8/pinky.ext/blob/master/background.js) if you want to know what needs to be redirected.

**Comment by Wicked-Mouse** [comment]
I'm more of a hobbyist programmer and my knowledge of javascript is pretty limited. I've looked at the js file and got as far as understanding that you made two constant arrays plus a constant to build the URLs out of a online library from ubisoft and replace each image with the same one from the same library.
What I was thinking was using the original images and add like a border to it, so that the collectible would  be visible, but still clearly stand out.
I'm also new on this website so I did not know/see whether this was discussed before or not. Sorry for the 'duplicate request' :)
